src/prefect/cli/cloud/webhook.py

```diff
@@ -0,0 +1,197 @@
+"""
+Command line interface for working with webhooks
+"""
+
+import typer
+from typing_extensions import Annotated
+from typing import Optional
+
+from prefect.cli._types import PrefectTyper
+from prefect.cli.cloud.cloud import cloud_app, confirm_logged_in
+from prefect.client.cloud import get_cloud_client
+from prefect.cli._utilities import exit_with_error
+from prefect.cli.root import app
+from prefect.settings import PREFECT_API_URL
+from rich.table import Table
+
+webhook_app = PrefectTyper(
+    name="webhook", help="Commands for interacting with Prefect Cloud Webhooks"
+)
+cloud_app.add_typer(webhook_app, aliases=["webhooks"])
+
+
+@webhook_app.command()
+async def get(
+    webhook_id: Annotated[Optional[str], typer.Argument()] = None,
+    all: bool = typer.Option(
+        False, "--all", "-a", help="Retrieve all webhooks in your workspace"
+    ),
+):
+    """
+    Retrieve a webhook by ID.
+    Optionally, fetch all webhooks with the --all flag
+    """
+
+    if not webhook_id and not all:
+        exit_with_error("Please provide a webhook ID or use the --all flag")
+
+    if webhook_id and all:
+        exit_with_error(
+            "Please provide a webhook ID or use the --all flag, but not both"
+        )
+
+    confirm_logged_in()
```

Comment:
It feels like there must be a way to add this as a hook for all the commands in this module, but I'm not sure how

Reply:
yeah it's a tedious without some automagic hook, but i couldn't find anything useful in the typer docs. fortunately, the hard part of making this into a util was already done 😄